procedures/katello/branch.md.erb

```diff
@@ -29,6 +29,12 @@
 ## Release Packager
 
 - [ ] Ensure tool_belt config is merged and output from `./tools.rb koji configs/katello/<%= release %>.yaml` matches expectations
+- [ ] Ensure releases of installer modules, this is typically done as part of Foreman module release workflow
+    - [ ] [candlepin](https://github.com/theforeman/puppet-candlepin)
+    - [ ] [pulp](https://github.com/theforeman/puppet-pulp)
+    - [ ] [pulpcore](https://github.com/theforeman/puppet-pulpcore)
+    - [ ] [foreman_proxy_content](https://github.com/theforeman/puppet-foreman_proxy_content)
+    - [ ] [katello](https://github.com/theforeman/puppet-katello)
```

Comment:
Do we need to name every one? We do it as part of the Foreman Installer release so really the message should be that all (major) changes should be in 2 weeks before the branching happens.

Reply:
AFAIK there is no reason to list each one. Thanks!